pages/common/dep.md

```diff
@@ -1,24 +1,33 @@
 # dep
 
-> Tool for dependency management in Go projects.
-> More information: <https://golang.github.io/dep>.
+> A CLI tool for deployment of PHP applications.
+> The goland command `dep` with the same name is deprecated and archived.
```

Comment:
I'd suggest adding some prefix to this sentence since it's not related to the actual tool being documented. Also, IIUC there's a typo here, in "goland" (cc @navarroaxel):

```suggestion
> Note: The golang command `dep` with the same name is deprecated and archived.
```

Reply:
It should be called "The Go command" if we want to be technically correct :)